zk-sdk/src/range_proof/generators.rs

```diff
@@ -0,0 +1,156 @@
+use {
+    crate::range_proof::errors::RangeProofGeneratorError,
+    curve25519_dalek::{
+        digest::{ExtendableOutput, Update, XofReader},
+        ristretto::RistrettoPoint,
+    },
+    sha3::{Sha3XofReader, Shake256},
+};
+
+#[cfg(not(target_os = "solana"))]
+const MAX_GENERATOR_LENGTH: usize = u32::MAX as usize;
```

Comment:
Is this meant to have the `#[cfg(...)]` here? It's used later in code that's not gated with the same cfg